Modules/SubModule_Import/xASL_bids_BIDS2Legacy.m

```diff
@@ -10,7 +10,7 @@
 %   dataPar    - dataPar values to be filled in a basic dataPar created with the conversion to legacy 
 %                (OPTIONAL, DEFAULT = basic dataPar settings)
 %   
-% OUTPUT: n/a
+% OUTPUT: dataPar - dataPar struct (STRUCT)
```

Comment:
Line 4 - FORMAT - x missing

Reply:
Oh sorry, I was updating the IO parameters multiple times and forgot to update the format some of the times. Should be fixed in 77a1e39 👍